src/playground/components/RuleItem.js

```diff
@@ -1,16 +1,11 @@
-import React, { useRef, useEffect } from "react";
+import React from "react";
 
-export default function RuleItem({ children }) {
-    const ref = useRef();
-
-    useEffect(() => {
-        ref.current.scrollIntoView({ behavior: "smooth", block: "end" });
-        ref.current.children[1].focus();
-    }, []);
-
-    return (
-        <li ref={ref} className="config__added-rules__item">
+const RuleItem = ({ children }) =>
+    (
+        <li className="config__added-rules__item">
             {children}
         </li>
     );
-}
+
+RuleItem.displayName = "RuleItem";
```

Comment:
```suggestion
```
Do we need this? We are already exporting an method devtools should pick its name?

Reply:
This was needed when `forwardRef` was used. Removed it now